.nancy-ignore

```diff
@@ -1,2 +1,5 @@
 CVE-2022-29153
 sonatype-2022-6522
+
+# Ignoring pkg:golang/golang.org/x/net@v0.5.0 (indirect dependency)
+CVE-2022-41723 until=2023-06-25
```

Comment:
Github is warning you that you are missing a new line at the end of the file https://stackoverflow.com/a/729795/563072